from lib_shared.core.utils.invalid_type_exception import InvalidTypeException


def test_works() -> None:
    assert 1 == 1
